include/deal.II/physics/transformations.h

```diff
@@ -789,6 +857,55 @@ namespace internal
 
 
 
+template<typename Number>
+Tensor<2,2,Number>
+Physics::Transformations::Angular::rotation_matrix_2d (const Number &angle)
+{
+  return Tensor<2,2> (
+  {
+    {
```

Comment:
Ok thanks. I was trying to avoid the creation of temporary data, but if its not C++98 compatible I'll make the change.